package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -345,53 +447,79 @@ def read_DATA_timestep(self, n_atoms, TS_class, TS_kwargs,
 
         return ts
 
-    def _parse_pos(self, datalines):
-        """Strip coordinate info into np array"""
-        pos = np.zeros((len(datalines), 3), dtype=np.float32)
+    def _parse_pos(self, datalines, header):
+        """Strip coordinate info into a np array
+
+        Arguments
+        ---------
+        datalines : list of str
+            list of strings (raw lines) from data file.
+        header: dict of str
+            A dict in which a key and value pair is a 'header' name and
+            its value in str foramt.
+
+        Returns
+        -------
+        pos: np.ndarray
+            array of (x,y,z) coordinates of atoms in a data file.
+        order : np.ndarray
+            array of integers (resulted from applying np.argsort on atom ids
+            that corrrectly maps the velocities into their corrresponding atom
```

Comment:
Not always velocities/coords that need to be mapped, clarify in docs.

Reply:
Read the lines I added to the doc.